tests/dara_collectors/test_dummy_dynamics_data_collector.py

```diff
@@ -0,0 +1,27 @@
+import pytest
+import torch
+
+from src.data_collectors.dummy_dynamics_data_collector import DummyDynamicsDataCollector
+from src.utils.step_record import RecordKeys as RK
+
+
+@pytest.mark.parametrize(
+    """
+    action_shape,
+    observation_shape,
+    get_size,
+    """,
+    [
+        ((4, 1), (623,), 3),
+        ((32,), (5, 64), 4),
+        ((17, 4, 7), (1, 3, 234), 5),
+    ],
+)
+def test_dummy_dynamics_data_collector(action_shape, observation_shape, get_size):
+    dummy_dynamics_data_collector = DummyDynamicsDataCollector(action_shape, observation_shape, get_size)
+
+    prev_actions, observations, actions, next_observations = dummy_dynamics_data_collector.get_data()[0:get_size]
```

Comment:
正確に`get_size`分取れてるか修正
```suggestion
    prev_actions, observations, actions, next_observations = dummy_dynamics_data_collector.get_data()[:]
```